packages/app/src/static-props/variants/get-variant-table-data.ts

```diff
@@ -18,13 +20,19 @@ export type VariantRow = {
   color: string;
 };
 
+export type VariantTableData = ReturnType<typeof getVariantTableData>;
+
 export function getVariantTableData(
-  nlVariants: NlVariants | undefined,
+  nlVariants: NlVariants | InVariants | undefined,
```

Comment:
Hehe here the prefix is biting your ass

```suggestion
  variants?: NlVariants | InVariants | undefined,
```